docs/personas.md

```diff
@@ -0,0 +1,7 @@
+**Andrew**: A non technical person who needs to see the overall health of the project. Andrew is currently only judged by his boss on shipping features, which are easily measurable based on completed stories in JIRA. This report exists partially to create quantitative metrics about quality for him to be judged on.
+
+**Barbara**: Barbara is a core web engineer who is savvy with both the report card and the tools it uses under the hood. She will use the information to both help justify their ongoing costs and to help plan where the engineering and advocacy efforts need to be focused. Barbara will be focused more on the reports cards efficacy and advocating its use than consuming the report directly since ideally she will not be in a position to call somebody out for having a bad report.
```

Comment:
True story, I didn't think about that. It could roughly translate to tech/project lead or engineering manager.